src/main/java/com/faforever/client/chat/PircBotXChatService.java

```diff
@@ -413,6 +420,12 @@ private void onPrivateMessage(PrivateMessageEvent event) {
       return;
     }
     logger.debug("Received private message: {}", event);
+
+    Player sourcePlayer = playerService.getPlayerForUsername(user.getNick());
+    if (sourcePlayer != null && sourcePlayer.getSocialStatus() == SocialStatus.FOE && preferencesService.getPreferences().getChat().getHideFoeMessages()) {
+      log.info("Chat message was suppressed as sender was foe, sender was: {}", user.getNick());
```

Comment:
Please log as `debug`
And while we're at it, we can shorten it and streamline the wording to:
`"Suppressing chat message from foe '{}'"`